src/app/app.component.html

```diff
@@ -12,4 +12,8 @@ <h5>Turntabl</h5>
     </div>
   </nav>
 </div>
+<div class="lds-ripple">
```

Comment:
Eeerrrmmm... do these divs have any significant functionalities?

Reply:
naa you can delete it...it is a dead code